extensions/iidm/resources/loadDetailRef.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_2" xmlns:ld="http://www.itesla_project.eu/schema/iidm/ext/load_detail/1_0" id="test" caseDate="2016-06-27T12:27:58.535000+02:00" forecastDistance="0" sourceFormat="test">
```

Comment:
http://www.powsybl.org/schema/iidm/1_3

Reply:
This branch has been rebased to retrieve version number update (07d76343a40be410c17e90abb19fb0cfb4370e47)
=> fixed